Form-Controls/index.html

```diff
@@ -13,15 +13,56 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <fieldset> 
+          <legend>CUSTOMER INFORMATION </legend>
+            <div> 
+              <label for='name'>Name:</label><br>
+              <input type='text' name='name' id='name' required minlength='2'>
+    
+            </div>
+            <div> 
+              <label for='email'> Email:</label><br>
+              <input type='email' name='email' id='email' required>
+            </div>  
+            
+        
+        </fieldset>
+        <fieldset>
+          <legend>PRODUCT INFORMATION</legend>
+          <div> 
+            <label for='color'>T-shirt color</label><br>
+            <select id ='color' name='tshirt_color' required>
+              <option value="red">red</option>
+              <option value="blue">blue</option>
+              <option value="purple">purple</option>
+            </select>  
+  
+          </div>
+          <div> 
+            <label for='size'>Size:</label><br>
+            <select id ='size' name='size' required>
```

Comment:
If I forget to choose a size, I'll order a size XS t-shirt! Can you think of a better user experience for this drop down?